client/src/services/token.ts

```diff
@@ -1,27 +1,18 @@
-import { graphLinks } from '../graphHelper'
-
 //TODO: Make this context rather than class
 export class TokenService {
     private static token = localStorage.getItem('token') ?? ''
-    private static expirationDate = new Date(localStorage.getItem('expirationDate') as string) ?? new Date()
-    static timeToExpire: number = 60 * 60 * 1000 - 100 // 1 hour - 100ms buffer
-
-    public static async getToken() {
```

Comment:
Token need to be checked everytime you make a call still, otherwise if you go to https://pr-pipline-features-aut-0b6ny4.herokuapp.com/users and you haven't logged before / or expired token you get a blank screen

Reply:
I fixed it in middleware, in the mw we are checking

- Is there token?
- If yes check if it is expired
- If yes make a refresh call and set cookie

I made silly mistake of opposite check, just commited change which implements above logic, hence the refresh call will be made only when token has expired.